src/main/java/cz/geokuk/plugins/mrizky/JMrizkaDdMmMmm.java

```diff
@@ -22,12 +22,12 @@ public JSingleSlide0 createRenderableSlide() {
 
 	@Override
 	public String getTextX(final double x) {
-		return Wgs.toGeoFormat(x);
+		return (x < 0 ? "-" : "") + Wgs.toGeoFormat(Math.abs(x));
```

Comment:
Minus logika patri asi spis do Wgs, ale pokud to rozbije neco jinde, nechme to takhle.